paasta_tools/kubernetes/bin/paasta_cleanup_stale_nodes.py

```diff
@@ -59,7 +59,7 @@ def nodes_for_cleanup(ec2_client: Client, nodes: Sequence[V1Node]) -> List[V1Nod
         node
         for node in nodes
         if not is_node_ready(node)
-        and "node-role.kubernetes.io/master" not in node.metadata.labels
+        and "node-role.kubernetes.io/control-plane" not in node.metadata.labels
```

Comment:
Do all our master nodes have control-plane label right now?

Reply:
I'm checking this right, now, in prod we do